lib/browser/api/extensions.js

```diff
@@ -538,7 +538,13 @@ ipcMain.on('chrome-cookies-getall', function (evt, responseId, details) {
 ipcMain.on('chrome-cookies-get', function (evt, responseId, details) {
   const cb = (error, cookie) => {
     if (!evt.sender.isDestroyed()) {
-      evt.sender.send('chrome-cookies-get-response-' + responseId, error, cookie)
+      var first = cookie
```

Comment:
why does `Cookies::GetAll` exist? It just calls `Cookies::Get` which seems to make for a confusing api
As far as this change goes, it doesn't seem to properly implement the api  from the docs at https://developer.chrome.com/extensions/cookies#method-get
"the one with the longest path will be returned"